ligo/test/Test/Ligo/BaseDAO/Token.hs

```diff
@@ -0,0 +1,44 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.Ligo.BaseDAO.Token
+  ( test_BaseDAO_Token
+  ) where
+
+import Universum
+
+import Morley.Nettest
+import Morley.Nettest.Tasty (nettestScenario)
+import Test.Tasty (TestTree, testGroup)
+
+import qualified BaseDAO.ShareTest.Token as Share
+import qualified Lorentz.Contracts.BaseDAO.Types as DAO
+import Test.Ligo.BaseDAO.Common
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
+
+test_BaseDAO_Token :: TestTree
+test_BaseDAO_Token = testGroup "BaseDAO non-FA2 token tests:"
+  [ testGroup "Admin:"
+    [ nettestScenario "can burn tokens from any accounts"
+        $ uncapsNettest $ Share.adminBurnScenario False
+          $ originateLigoDaoWithBalance
+            (\o1 _ ->
+                [ ((o1, DAO.unfrozenTokenId), 10)
+                , ((o1, DAO.frozenTokenId), 10)
+                ]
+            )
+    , nettestScenario "can mint tokens to any accounts"
+        $ uncapsNettest $ Share.adminMintScenario False
+          $ originateLigoDaoWithBalance
+            (\o1 _ ->
+                [ ((o1, DAO.unfrozenTokenId), 0)
+                , ((o1, DAO.frozenTokenId), 0)
+                ]
+            )
+    ]
+  , testGroup "Other:"
+    [ nettestScenario "can call transfer tokens entrypoint"
```

Comment:
I figure this does no longer need to stay in a different test group, now that we know the admin constraint is here too.

In fact the previous 2 tests shouldn't be in an "Admin:" test group either because they also check for non-admin.
I think we should just use the topmost `testGroup "BaseDAO non-FA2 token tests:"` here.

Reply:
Make sense, I've updated this, as well as other changes.